source/_UIAHandler.py

```diff
@@ -377,21 +374,7 @@ def _isUIAWindowHelper(self,hwnd):
 				# Allow the user to explisitly force UIA support for MS Word documents no matter the Office version 
 				and not config.conf['UIA']['useInMSWordWhenAvailable']
 			):
-				# We can only safely check the version of known Office apps using the Word document control, as we know their versioning scheme.
-				# But if the Word Document control is used in other unknown apps we should just use UIA if it has been implemented.
-				if appModule.appName not in ('outlook','winword','excel'):
-					log.debugWarning("Unknown application using MS Word document control: %s"%appModule.appName)
-					return True
-				try:
-					versionMajor,versionMinor,versionBuild,versionPatch=[int(x) for x in appModule.productVersion.split('.')]
-				except Exception as e:
-					log.error("Error parsing version information %s, %s"%(appModule.productVersion,e))
-					return True
-				if (
-					versionMajor<minMSWordUIAVersion.major
-					or versionMajor==minMSWordUIAVersion.major and versionMinor==minMSWordUIAVersion.minor and versionBuild<minMSWordUIAVersion.build

```

Comment:
I think we should leave in place checks that exclude versions that we know will not work. Though this should be reflected in the user guide; "This setting will not enable UIA in MS Word versions below 16.0.9000"